<?xml version="1.0" ?>
<testsuite errors="0" failures="0" name="unittest.test.testmock.testhelpers.TestCallList-20200422102217" tests="4" time="0.001">
	<testcase classname="unittest.test.testmock.testhelpers.TestCallList" name="test_args_list_contains_call_list" time="0.000"/>
	<testcase classname="unittest.test.testmock.testhelpers.TestCallList" name="test_call_list_str" time="0.000"/>
	<testcase classname="unittest.test.testmock.testhelpers.TestCallList" name="test_propertymock" time="0.000"/>
	<testcase classname="unittest.test.testmock.testhelpers.TestCallList" name="test_propertymock_returnvalue" time="0.000"/>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
